enterprise/cmd/frontend/internal/auth/gitlaboauth/config.go

```diff
@@ -49,7 +50,7 @@ type Provider struct {
 	providers.Provider
 }
 
-func parseConfig(cfg conftypes.SiteConfigQuerier, db database.DB) (ps []Provider, problems conf.Problems) {
+func parseConfig(cfg conftypes.SiteConfigQuerier, logger log.Logger, db database.DB) (ps []Provider, problems conf.Problems) {
```

Comment:
Logger should be the first parameter:

```suggestion
func parseConfig(logger log.Logger, cfg conftypes.SiteConfigQuerier, db database.DB) (ps []Provider, problems conf.Problems) {
```

Reply:
aaaah I was unsure about this! Thanks :D